boms/cloud-lts-bom/pom.xml

```diff
@@ -337,6 +333,7 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
+        <!-- Not using a BOM because it has google-cloud-bigtable-emulator -->
```

Comment:
@kolea2 FYI, I added comment about not using google-cloud-bigtable-bom.

Reply:
Thanks!